gensim/models/phrases.py

```diff
@@ -473,22 +570,27 @@ def export_phrases(self, sentences, out_delimiter=b' ', as_tuples=False):
                     yield (out_delimiter.join(words), score)
 
     def __getitem__(self, sentence):
-        """
-        Convert the input tokens `sentence` (=list of unicode strings) into phrase
+        """Convert the input tokens `sentence` (=list of unicode strings) into phrase
         tokens (=list of unicode strings, where detected phrases are joined by u'_').
 
         If `sentence` is an entire corpus (iterable of sentences rather than a single
         sentence), return an iterable that converts each of the corpus' sentences
         into phrases on the fly, one after another.
 
-        Example::
-
-          >>> sentences = Text8Corpus(path_to_corpus)
-          >>> bigram = Phrases(sentences, min_count=5, threshold=100)
-          >>> for sentence in phrases[sentences]:
-          ...     print(u' '.join(s))
-            he refuted nechaev other anarchists sometimes identified as pacifist anarchists advocated complete
-            nonviolence leo_tolstoy
+        Parameters
+        ----------
+        sentence : list of str
+            List of unicode strings.
+
+        Example
+        ----------
+        >>> from gensim.test.utils import datapath
+        >>> from gensim.models.word2vec import Text8Corpus
+        >>> from gensim.models.phrases import Phrases
+        >>> sentences = Text8Corpus(datapath('testcorpus.txt'))
+        >>> bigram = Phrases(sentences, min_count=5, threshold=100)
+        >>> bigram["trees","grass"]
```

Comment:
list of .., i.e. should be `bigram[["trees", "grass"]]`, also, this have sense to make example that produced bigram